common/source/docs/common-fettec-onewire.rst

```diff
@@ -1,8 +1,13 @@
-==============
-FETtec OneWire
-==============
+===================
+FETtec OneWire ESCs
+===================
 
-This is a `half duplex serial protocol with a 2Mbit/s Baudrate<https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_FETtec_OneWire>`_ created by Felix Niessen (former Flyduino KISS developer) from FETtec. It supports ESC telemetry, so information from the ESC status is sent back to the autopilot:
+.. note::
```

Comment:
we dont do this...if it will be in master only, we will just add this page to Upcoming features....

Reply:
I hope this will be released at some point, and users of 4.0 should not be surprised if this does not work for them